fspiop-api/documents/API-Definition_v1.1.1.md

```diff
@@ -2183,7 +2183,7 @@ Table 29 contains a description of each different version of the **/transfers**
 | ---- | ---- | ---- |
 | **1.0** | 2018-03-13 | Initial version |
 | **1.1** | 2020-05-19 | The resource is updated to support commit notifications using HTTP Method **PATCH**. The new request **PATCH /transfers/{ID}** is described in Section 6.7.3.3. The process of using commit notifications is described in Section 6.7.2.6. <br><br> The data model is updated to add an optional ExtensionList element to the PartyIdInfo complex type based on the Change Request: [https://github.com/mojaloop/mojaloop-specification/issues/30](https://github.com/mojaloop/mojaloop-specification/issues/30). Following this, the data model as specified in Table 93 has been updated.|
-| **1.2** | 2022-01-31 | The resource is extended to support commit notifications for failures, using the HTTP Method **PATCH** introduced in version **1.1**. This can be used when there is an internal validation or processing error (not schema validation) during handling of the **PUT /transfers/**_{ID}_ sent from the Payee FSP, both when RESERVED and COMMITTED is used as transferState. The new request **PATCH /transfers/{ID}/error** is described in Section 6.7.5.2. The process of using commit notifications is described in Section 6.7.2.6. <br><br> This is based on the acceptance of the change request: https://github.com/mojaloop/mojaloop-specification/issues/103|
+| **1.2** | 2022-01-31 | The resource is extended to support commit notifications for failures, using the HTTP Method **PATCH** introduced in version **1.1**. This can be used when there is an internal validation or processing error (not schema validation) during handling of the **PUT /transfers/**_{ID}_ sent from the Payee FSP, when COMMITTED is used as transferState. The new error callback **PATCH /transfers/{ID}/error** is described in Section 6.7.5.2. <br><br> This is based on the acceptance of the change request: https://github.com/mojaloop/mojaloop-specification/issues/103|
```

Comment:
It shouldn't be a commit notification, it is an error notification for commit failures. Commit notification is using **PATCH /transfers/**{ID}, error notifications is using **PATCH /transfers/**{ID}**/error.**

Similarly, "error callback" should be "error notification". PUTs are used for callbacks, PATCH for notifications.

Minor note is that we are not using the line breaks (&lt;br>&lt;br>) elsewhere before linking to the change request